### PR TITLE
Incrementally better guidance on env vars

### PR DESCRIPTION
**Pull request by kcussen**
This PR is poorly named and is being made late at night after watching a sick toddler for a week. Kindly ensure I haven't given away any sensitive keys!

Changes:
- Commented where certain sensitive keys can be found for both api and webapp 
- Grouped keys from like providers
- Removed a few duplicate keys

### CODE REVIEW

File: packages/api/.env.default [hunk 1]
```diff
@@ -21,33 +21,41 @@ JWT_RESET_SECRET=Production_is_secured_with_a_long_random_string
...
+# Same as the VITE_GOOGLE_MAPS_API_KEY under webapp
 GOOGLE_API_KEY=?
 
-GMAIL_API_CLIENT_ID=?
-GMAIL_API_CLIENT_SECRET=?
-GMAIL_ACCESS_TOKEN=?
-GMAIL_REFRESH_TOKEN=?
+# Create your own free or student OpenWeather API key at https://openweathermap.org/price
+# Same as the VITE_WEATHER_API_KEY under webapp
+OPEN_WEATHER_APP_ID=?
```
Comment (Duncan-Brain) [inline packages/api/.env.default:30]: Were there any limitations on this public key? I thought someone mentioned that there might be for some features?

File: packages/api/.env.default [hunk 2]
```diff
@@ -21,33 +21,41 @@ JWT_RESET_SECRET=Production_is_secured_with_a_long_random_string
...
 
+# Create your own Google API key at https://developers.google.com/identity/oauth2/web/guides/get-google-api-clientid
 GMAIL_API_CLIENT_ID=?
 GMAIL_API_CLIENT_SECRET=?
+GMAIL_ACCESS_TOKEN=?
```
Comment (Duncan-Brain) [inline packages/api/.env.default:35]: Did we ever find out where GMAIL items came from? I still feel like we haven't found where these are at.

**Comment by Duncan-Brain** [comment]
@kcussen 

[Jira Issue LF-2867 ](https://lite-farm.atlassian.net/browse/LF-2867?atlOrigin=eyJpIjoiMDk1OWYyYTgxMzNiNDQwNWI1Y2UxZTIzYTA4MjNmMWYiLCJwIjoiaiJ9)

1. Giving away secrets - if any of these are on prod or beta with the same (however trivial) value as a pass, secret, key, token. Yeah you may want to change that.
2. My personal opinion is that grouping of vars by necessary-ness for Local, Server(Prod/Beta), and Experimental would be easiest to follow for contributors.
3. I have added some inline comments about a couple that I am still unsure about.

**Review by anant00729 — APPROVED** [review]

File: packages/api/.env.default (hunk 2 above)
Comment (kcussen) [reply packages/api/.env.default:35]: No, I poked around and don't know where they came from. We also had them duplicated.

Comment (Duncan-Brain) [reply packages/api/.env.default:35]: Is there anyone we can ask about them?

**Pull request merged**